tests/expected-readme.md

```diff
@@ -290,3 +291,9 @@ Helm performs a lookup for the object based on its group (apps), version (v1), a
 In https://github.com/helm/charts/pull/17285 the `apiVersion` of the deployment resources was updated to `apps/v1` in tune with the api's deprecated, resulting in compatibility breakage.
 
 This major version signifies this change.
+
+## Test supporting data
+
+### Additional square brackets
+
+Additional square brackets section of the readme. Which can be linked to from injected Parameters section.
```

Comment:
Delete this, it is not necessary to edit this file since it is a Helm Chart README example to test the tool works as expected, so it is not intended to be edited for providing information to users.

Reply:
Removed redundant text